<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addMusicButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>从文件添加歌曲</string>
    </property>
   </widget>
   <widget class="QLabel" name="musicLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>541</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="musicpositionSlider">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>571</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="musictimeLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>390</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playBackMode_pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>420</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="musicPlayerImageSources/iconSources.qrc">
      <normaloff>:/LoopIcon.png</normaloff>:/LoopIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>58</width>
      <height>58</height>
     </size>
    </property>
   </widget>
   <widget class="QListWidget" name="playlist_listWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lyrics_listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>381</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="background_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>801</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="musicPlayerImageSources/iconSources.qrc">:/background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clearList_pushButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>清空播放列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lastmusicpushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>420</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="musicPlayerImageSources/iconSources.qrc">
      <normaloff>:/lastmusicIcon.png</normaloff>:/lastmusicIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>58</width>
      <height>58</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="playpushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>420</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="musicPlayerImageSources/iconSources.qrc">
      <normaloff>:/pauseIcon.png</normaloff>:/pauseIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>58</width>
      <height>58</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="nextmusicpushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>420</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="musicPlayerImageSources/iconSources.qrc">
      <normaloff>:/nextmusicIcon.png</normaloff>:/nextmusicIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>58</width>
      <height>58</height>
     </size>
    </property>
   </widget>
   <zorder>addMusicButton</zorder>
   <zorder>playlist_listWidget</zorder>
   <zorder>lyrics_listWidget</zorder>
   <zorder>background_label</zorder>
   <zorder>playBackMode_pushButton</zorder>
   <zorder>lastmusicpushButton</zorder>
   <zorder>playpushButton</zorder>
   <zorder>nextmusicpushButton</zorder>
   <zorder>musicpositionSlider</zorder>
   <zorder>musictimeLabel</zorder>
   <zorder>musicLabel</zorder>
   <zorder>clearList_pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="musicPlayerImageSources/iconSources.qrc"/>
 </resources>
 <connections/>
</ui>
